### PR TITLE
Add operators that are missing on the detail page

### PR DESCRIPTION
**Pull request by pontusmelke**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-839.surge.sh

Updated pages:
- [Operators in detail](https://neo4j-docs-cypher-839.surge.sh/cypher-manual/5/planning-and-tuning/operators/operators-detail)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-839.surge.sh

Updated pages:
- [Operators in detail](https://neo4j-docs-cypher-839.surge.sh/cypher-manual/5/planning-and-tuning/operators/operators-detail)

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 1]
```diff
@@ -1064,12 +1148,57 @@ Total database accesses: 1, total allocated memory: 184
 == Node By Id Seek
 // NodeByIdSeek
 
-The `NodeByIdSeek` operator reads one or more nodes by ID from the node store, specified via the function xref::functions/scalar.adoc#functions-elementid[elementId()].
+The `NodeByIdSeek` operator reads one or more nodes by ID from the node store, specified via the function xref::functions/scalar.adoc#functions-id[id()].
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
The `NodeByIdSeek` operator reads one or more nodes by id from the node store, specified via the function xref::functions/scalar.adoc#functions-id[id()].
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 2]
```diff
@@ -1064,12 +1148,57 @@ Total database accesses: 1, total allocated memory: 184
...
+1 row
+----
+
+======
+
+[[query-plan-node-by-element-id-seek]]
+== Node By Element Id Seek
+// NodeByElementIdSeek
+
+The `NodeByElementIdSeek` operator reads one or more nodes by element ID from the node store, specified via the function xref::functions/scalar.adoc#functions-elementid[elementId()].
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
The `NodeByElementIdSeek` operator reads one or more nodes by element id from the node store, specified via the function xref::functions/scalar.adoc#functions-elementid[elementId()].
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 3]
```diff
@@ -2873,6 +3054,59 @@ Total database accesses: 318, total allocated memory: 208
 
 ======
 
+[[query-plan-nullify-metadata]]
+== Nullify Metadata
+// NullifyMetadata
+_This feature was introduced in Neo4j 5.9._
+NullifyMetadata is responsible for cleaning up state associated with Repeat(Trail), it is only planned directly after Repeat(Trail).
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]: Im not sure if my suggestion to add a "the" before "state" is correct here. In any case, can we explain briefly what the "state" refers to here?

```suggestion
_This feature was introduced in Neo4j 5.9._

`NullifyMetadata` is responsible for cleaning up the state associated with `Repeat(Trail)`.
It is only planned directly after `Repeat(Trail)`.
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 4]
```diff
@@ -3583,6 +3816,110 @@ Total database accesses: 9, total allocated memory: 64
 
 ======
 
+[query-plan-transaction-foreach]]
+== TransactionForeach
+// TransactionForeach
+
+`TransactionForeach` works like normal `Foreach` but will commit the current transaction after a specified number of rows.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
`TransactionForeach` works like the `Foreach` operator but will commit the current transaction after a specified number of rows.
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 5]
```diff
@@ -3583,6 +3816,110 @@ Total database accesses: 9, total allocated memory: 64
...
+ Total database accesses: 12, total allocated memory: 5704
+----
+
+======
+
+[query-plan-subquery-foreach]]
+== SubqueryForeach
+// SubqueryForeach
+
+`SubqueryForeach` like normal `Foreach` but used for executing sub-queries.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
`SubqueryForeach` works like the `Foreach` operator but it is only used for executing subqueries.
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 6]
```diff
@@ -5713,3 +6095,64 @@ Total database accesses: 0, total allocated memory: 64
 
 ======
 
+[[query-plan-argument-tracker]]
+== Argument Tracker
+// ArgumentTracker
+
+The `ArgumentTracker` operator is used to make sure we have row-by-row semantics.
+This restricts the runtime not to batch operations in larger chunks.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
The `ArgumentTracker` operator is used to ensure row-by-row semantics.
This restricts the xref:planning-and-tuning/runtimes/index.adoc[Cypher runtime] to not batch operations in larger chunks.
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 7]
```diff
@@ -311,6 +311,49 @@ Total database accesses: 2, total allocated memory: 184
 ======
 
 
+[[query-plan-directed-relationship-by-element-id-seek]]
+== Directed Relationship By Element Id Seek
+// DirectedRelationshipByElementIdSeek
+
+The `DirectedRelationshipByElementIdSeek` operator reads one or more relationships by element id from the relationship store, and produces both the relationship and the nodes on either side.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]: What is meant by "on either side" here? Do you mean "and the nodes on either side of the relationship" or something else?

```suggestion
The `DirectedRelationshipByElementIdSeek` operator reads one or more relationships by element id from the relationship store (specified via the function xref::functions/scalar.adoc#functions-elementid[elementId()]) and produces both the relationship and the nodes on either side.
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 8]
```diff
@@ -341,31 +384,28 @@ Runtime version {neo4j-version-minor}
...
+[[query-plan-undirected-relationship-by-element-id-seek]]
+== Undirected Relationship By Element Id Seek
+// UndirectedRelationshipByElementIdSeek
 
-[[query-plan-undirected-relationship-by-id-seek]]
-== Undirected Relationship By Id Seek
-// UndirectedRelationshipByIdSeek
-
-The `UndirectedRelationshipByIdSeek` operator reads one or more relationships by id from the relationship store.
+The `UndirectedRelationshipByElementIdSeek` operator reads one or more relationships by element id from the relationship store.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
The `UndirectedRelationshipByElementIdSeek` operator reads one or more relationships by element id from the relationship store (specified via the function xref::functions/scalar.adoc#functions-elementid[elementId()]).
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 9]
```diff
@@ -401,6 +441,50 @@ Total database accesses: 1, total allocated memory: 184
 
 ======
 
+[[query-plan-undirected-relationship-by-id-seek]]
+== Undirected Relationship By Id Seek
+// UndirectedRelationshipByIdSeek
+
+The `UndirectedRelationshipByIdSeek` operator reads one or more relationships by id from the relationship store.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
The `UndirectedRelationshipByIdSeek` operator reads one or more relationships by id from the relationship store (specified via the function xref::functions/scalar.adoc#functions-elementid[elementId()]).
```

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 10]
```diff
@@ -1768,6 +1897,58 @@ Total database accesses: 166, total allocated memory: 976
 
 ======
 
+[query-plan-transaction-apply]]
+== TransactionApply
+// TransactionApply
+
+`TransactionApply` works like normal `Apply` but will commit the current transaction after a specified number of rows.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
`TransactionApply` works like the `Apply` operator but will commit the current transaction after a specified number of rows.
```

**Review by JPryce-Aklundh — COMMENTED** [review]
Thanks Pontus - Just a few questions and editorial questions.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-839.surge.sh

Updated pages:
- [Operators in detail](https://neo4j-docs-cypher-839.surge.sh/cypher-manual/5/planning-and-tuning/operators/operators-detail)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by JPryce-Aklundh — APPROVED** [review]

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 11]
```diff
@@ -5713,3 +6097,64 @@ Total database accesses: 0, total allocated memory: 64
...
+| | |                +----+-------------------------------------------------------+----------------+------+---------+----------------+                        |           |                     |
+| | +Argument        |  5 | s                                                     |              0 |    0 |       0 |            256 |                    0/0 |     0.000 | Fused in Pipeline 1 |
+| |                  +----+-------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+| +NodeIndexSeek     |  6 | RANGE INDEX s:Person(name) WHERE name = $autostring_0 |              0 |    0 |       1 |            376 |                    1/0 |     0.286 | In Pipeline 0       |
++--------------------+----+-------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------------+
+
+Total database accesses: 1, total allocated memory: 2104
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]:
```suggestion
```

**Pull request merged**